backend/app/db/models/case.py

```diff
@@ -0,0 +1,145 @@
+        """Creates the enum from a string value."""
+        if value is None:
+            return None
+        if value.upper() == "LOSS AND DAMAGE":
+            return UNFCCCPillars.LOSS_DAMAGE
+        if value == "Mitigation and Adaptation & Resilience":
+            return UNFCCCPillars.ADAPTATION_RESILIENCE
+        return UNFCCCPillars(value)
+
+
+class CaseOutcome(str, enum.Enum):
+    """The outcome of a Case."""
+
+    ENHANCED_CLIMATE_ACTION = "Enhanced climate action"
+    NEUTRAL = "Neutral"
+    HINDERED_CLIMATE_ACTION = "Hindered climate action"
+    NA = "N/A"
+
+
+class ClimateAlignmentClass(str, enum.Enum):
+    """The class of climate alignment."""
+
+    CLIMATE_ALIGNED = "Climate aligned"
+    CLIMATE_NON_ALIGNED = "Climate non-aligned"
+    JUST_TRANSITION = "Just transition"
+
+    @staticmethod
+    def from_value(value: str) -> Union["ClimateAlignmentClass", None]:
+        """Creates the enum from a string value."""
+        if value is None:
+            return None
+        if value == "N/A":
+            return None
+        return ClimateAlignmentClass(value)
+
+
+class StrategicAlignmentClass(str, enum.Enum):
+    """The class of strategic alignment."""
+
+    ENFORCING_CLIMATE_STANDARDS = "Enforcing climate standards"
+    GOVERNMENT_FRAMEWORK = "Government framework"
+    PUBLIC_FINANCE = "Public finance"
+    FAILURE_TO_ADAPT = "Failure to adapt"
+    COMPENSATION = "Compensation"
+    CLIMATE_WASHING = "Climate-washing"
+    PERSONAL_RESPONSIBILITY = "Personal responsibility"
+    REGULATORY_POWERS = "Regulatory powers"
+    SLAPP = "Strategic litigation against public participation (SLAPP)"
+    JUST_TRANSITION = "Just transition"
+    CORPORATE_FRAMEWORK = "Corporate framework"
+    OTHER = "Other"
+    NA = "N/A"
+
+    @staticmethod
+    def from_value(value: str) -> Union["StrategicAlignmentClass", None]:
+        """Creates the enum from a string value."""
+        if value is None:
+            return None
+        if value == "Stranded assets":
+            return StrategicAlignmentClass.OTHER
+        if value.upper() == "JUST TRANSITION":
+            return StrategicAlignmentClass.JUST_TRANSITION
+        return StrategicAlignmentClass(value)
+
+
+class Case(Base):  # noqa: D101
+    """Database model for a Litigation case."""
+
+    __tablename__ = "lit_case"
+
+    id = Column(Integer, primary_key=True)
+    ext_id = Column(Text)
+    name = Column(Text, unique=True, nullable=False)
+    year = Column(Integer)
+    status = Column(Enum(CaseStatus))
+    outcome = Column(Enum(CaseOutcome))
+    objective = Column(Text)
+    pillars = Column(Enum(UNFCCCPillars))
+    summary = Column(Text)
+    reference = Column(Text)
+    case_class = Column(Text)
+    strategic = Column(Boolean)
+    climate_class = Column(Enum(ClimateAlignmentClass))
+    strategic_class = Column(Enum(StrategicAlignmentClass))
+    source = Column(Text)
+    keywords = Column(ARRAY(Text))
```

Comment:
Confirm with @joel-wright

Reply:
The model we've used for all this with the law/policy data is to have each one have it's own table, and to define join tables for each metadata-type that can have multiple values associated with a document. Metadata-types with strictly one value have up to this point been implemented as a FK into the appropriate table directly on the document model.